nixos/modules/services/backup/borgbackup.nix

```diff
@@ -66,6 +66,7 @@ let
       ${mkKeepArgs cfg} \
       ${optionalString (cfg.prune.prefix != null) "--glob-archives ${escapeShellArg "${cfg.prune.prefix}*"}"} \
       $extraPruneArgs
+    borg compact
```

Comment:
(Not a maintainer.) I believe you should pass [`$extraArgs`](https://github.com/NixOS/nixpkgs/blob/a575c243c23e2851b78c00e9fa245232926ec32f/nixos/modules/services/backup/borgbackup.nix#L584-L592) to stay consistent with the module's API.